tests/test_retry.py

```diff
@@ -32,158 +32,156 @@
+        )
 
-        my_callable = retry.retryable(mock_call, retry_options)
+        my_callable = retry.retryable(mock_func, retry_options)
+        result = my_callable(None)
 
-        self.assertEqual(my_callable(None), 1729)
-        self.assertEqual(to_attempt, mock_call.call_count)
+        self.assertEqual(result, 1729)
+        self.assertEqual(to_attempt, mock_func.call_count)
 
-    @mock.patch('google.gax.config.exc_to_code')
     @mock.patch('time.time')
-    def test_retryable_with_timeout(self, mock_time, mock_exc_to_code):
+    def test_retryable_with_timeout(self, mock_time):
         mock_time.return_value = 1
-        mock_exc_to_code.side_effect = lambda e: e.code
 
-        mock_call = mock.Mock()
-        mock_call.side_effect = [CustomException('', _FAKE_STATUS_CODE_1),
-                                 mock.DEFAULT]
-        mock_call.return_value = 1729
+        mock_func = mock.Mock()
+        mock_func.side_effect = [
+            errors.MockGrpcException(code=_FAKE_STATUS_CODE_1),
+            mock.DEFAULT,
+        ]
+        mock_func.return_value = 1729
 
         retry_options = RetryOptions(
             [_FAKE_STATUS_CODE_1],
-            BackoffSettings(0, 0, 0, 0, 0, 0, 0))
+            BackoffSettings(0, 0, 0, 0, 0, 0, 0),
+        )
 
-        my_callable = retry.retryable(mock_call, retry_options)
+        my_callable = retry.retryable(mock_func, retry_options)
 
         self.assertRaises(errors.RetryError, my_callable)
-        self.assertEqual(0, mock_call.call_count)
+        self.assertEqual(0, mock_func.call_count)
 
-    @mock.patch('google.gax.config.exc_to_code')
     @mock.patch('time.time')
-    def test_retryable_when_no_codes(self, mock_time, mock_exc_to_code):
+    def test_retryable_when_no_codes(self, mock_time):
         mock_time.return_value = 0
-        mock_exc_to_code.side_effect = lambda e: e.code
-
-        mock_call = mock.Mock()
-        mock_call.side_effect = [CustomException('', _FAKE_STATUS_CODE_1),
-                                 mock.DEFAULT]
-        mock_call.return_value = 1729
 
+        # Set up the mock function to raise an exception that is *not*
+        # an expected code.
+        mock_func = mock.Mock()
+        mock_func.side_effect = [
+            errors.MockGrpcException(code=_FAKE_STATUS_CODE_2),
+            mock.DEFAULT,
+        ]
+        mock_func.return_value = 1729
+
+        # Set the retry options not to actually honor any codes
+        # (thus, our code is not in the list).
         retry_options = RetryOptions(
             [],
-            BackoffSettings(0, 0, 0, 0, 0, 0, 1))
+            BackoffSettings(0, 0, 0, 0, 0, 0, 1),
+        )
 
-        my_callable = retry.retryable(mock_call, retry_options)
-
-        try:
+        # Create the callable and establish that we get a GaxError.
+        my_callable = retry.retryable(mock_func, retry_options)
+        with self.assertRaises(errors.GaxError):
```

Comment:
This is testing weaker condition than before, since we're no longer checking the cause of the GaxError.